lib/babe/slot.go

```diff
@@ -0,0 +1,55 @@
+package babe
+
+import (
+	"time"
+)
+
+// timeUntilNextSlotInNanos calculates, based on the current system time, the remainng
+// time to the next slot
```

Comment:
perhaps helpful to link the substrate code here? Don't have too, just a possible suggestion if you think it could be helpful